### ISSUE TITLE
Updater - Malfunction 

### ISSUE THREAD

**Issue by ChrisMoraitisrnd3** [ChrisMoraitisrnd3/MAIC-Issues#23]
With unstable internet connection we have many problems about the upDater.
If for some reason the connection breaks (when maic is updating) , the update will never installed

And the box remains with the same version for ever..

**Comment by outcast1000** [comment]
Αυτό ισχυει από ποιά version και πάνω?

**Comment by ChrisMoraitisrnd3** [comment]
Απ'ο ότι έχω εντύπωση γινόταν πάντα και πρέπει να γίνει επαναφορά για να συνέλθει (και να έχει ιντερνετ- όχι με διακοπές)

**Comment by themelisx** [comment]
Προφανώς είναι αυτό που έχουμε ξαναπεί και θεωρητικά ο Jim έχει λύσει.
Δηλαδή: Το Unity δεν έχει πάρει το onDownloadComplete για να δώσει σήμα στον updater για APK install.
Εφόσον στις νεότερες εκδόσεις το Unity καλεί την ApplyUpdates() ή την ApplyUpdate(int downloadID), μετά από κάθε CheckForUpdates(), τότε δεν θα υπάρχει πρόβλημα

**Comment by outcast1000** [comment]
@themelisx @jimarab Επιβεβαιώστε ότι δουλεύει πια

**Comment by ChrisMoraitisrnd3** [comment]
όχι, δεν έγινε. Σε μία συσκευή είχε φθάσει 3% , την σύνδεσα αλλού και συνέχισε ξανά από το 3%
Ενώ σε μία άλλη που κόλλησε από τη αρχή στο "παρακαλώ περιμ΄νετε" μετά δε πήρε ποτέ update
